abuild.in

```diff
@@ -1695,6 +1695,7 @@ default_dbg() {
 			mv "$ddbg_dstfile" "$ddbg_dstdir"
 			${CROSS_COMPILE}strip "$ddbg_srcfile"
 			[ -n "$XATTR" ] && { echo "$XATTR" | setfattr --restore=-; }
+			exit 0
```

Comment:
How about something more readable like:
````diff
diff --git a/abuild.in b/abuild.in
index 9edea57..22f5390 100644
--- a/abuild.in
+++ b/abuild.in
@@ -1694,7 +1694,9 @@ default_dbg() {
                        ${CROSS_COMPILE}objcopy --add-gnu-debuglink="$ddbg_dstfile" "$ddbg_srcdir/$ddbg_srcfile"
                        mv "$ddbg_dstfile" "$ddbg_dstdir"
                        ${CROSS_COMPILE}strip "$ddbg_srcfile"
-                       [ -n "$XATTR" ] && { echo "$XATTR" | setfattr --restore=-; }
+                       if [ -n "$XATTR" ]; then
+                               echo "$XATTR" | setfattr --restore=-
+                       fi
                )
        done
        return 0
````

Reply:
I would like to keep the `exit 0 `at the end of the sub-shell. The rest looks good to me.